signac_dashboard/views.py

```diff
@@ -45,16 +47,19 @@ def jobs_list(dashboard):
     if not jobs:
         flash("No jobs found.", "warning")
     g.jobs = dashboard._get_job_details(g.pagination.paginate(jobs))
-    project_title = dashboard.project.config.get("project", None)
-    g.title = f"{project_title}: Jobs" if project_title else "Jobs"
+    project_path = dashboard.project.config["project_dir"]
+    _, project_folder = split(project_path)
```

Comment:
Do you want `os.path.basename` instead?

Reply:
Also I’d prefer to call it a directory instead of a folder.

Reply:
Also open for discussion is what the title should actually say now!

Reply:
Discussed in dev meeting this week. We'll just say "signac dashboard" for now